bundles-maven-plugin/README.md

```diff
@@ -0,0 +1,89 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+# Apache Metron Bundle Maven Plugin
```

Comment:
So - I just want to be sure we all agree before I move things.  I'm OK moving things as nick suggests above -> metron-extensions goes under metron-platform and everything goes in there, except the archetypes which are still at root of platform

```
/metron-platform
------>/metron-extensions
----------->/bundles-lib
----------->/bundles-maven-plugin
----------->/metron-parser-extensions
------>/metron-maven-archetypes
----------->/metron-maven-parser-extension-archetype


```